.gitignore

```diff
@@ -0,0 +1 @@
+/venv/
```

Comment:
This .gitignore file is a special git file where we can list all the files and folders that git should ignore i.e. it shouldn't prompt us to commit/push those files and folders. I have added /venv/ in this file to ensure that git will ignore that folder.